osm2streets/src/pathfinding.rs

```diff
@@ -57,8 +56,7 @@ impl StreetNetwork {
             &graph,
             from,
             |i| i == to,
-            // TODO Expensive!
```

Comment:
This stopped being expensive with the last PR that stores a polyline instead of vec<pt2d>